### PR TITLE
fix(files): prevent folder name of pure spaces

### PR DESCRIPTION
**Pull request by josephmcg**
**What this PR does** 📖
title

**Which issue(s) this PR fixes** 🔨
AP-1108

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by stavares843 — APPROVED** [review]

File: components/views/files/controls/Controls.vue [hunk 1]
```diff
@@ -61,8 +61,10 @@ export default Vue.extend({
      */
     async addFolder() {
       this.errors = []
-      if (!this.text) {
+      // checks for empty input OR input of only spaces
+      if (!this.text || this.text.replace(/\s/g, '').length === 0) {
```
Comment (WanderingHogan) [inline components/views/files/controls/Controls.vue:65]: can you make this `!RegExp(this.$Config.regex.<whatever-you-name-it>).test(this.text)`

Comment (WanderingHogan) [reply components/views/files/controls/Controls.vue:65]: or actually this wouldn't work for empty input

Comment (WanderingHogan) [reply components/views/files/controls/Controls.vue:65]: maybe `RegExp(/^\s*$/).test(this.text)`

Comment (josephmcg) [reply components/views/files/controls/Controls.vue]: thanks, added global for it

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**